pkg/controllers/machinedisruptionbudget/machinedisruptionbudget_controller.go

```diff
@@ -543,3 +544,30 @@ func RetryDecrementMachineDisruptionsAllowed(c client.Client, machine *mapiv1.Ma
 
 	return err
 }
+
+// isMachineHealthy returns true if the the machine is running and machine node is healthy
+func isMachineHealthy(c client.Client, machine *mapiv1.Machine) (bool, error) {
```

Comment:
can we pass `unhealtyConditions` to it once, instead of fetching it for each machine

Reply:
I put this one into separate commit https://github.com/kubevirt/machine-remediation-operator/pull/46/commits/65453f71190b9efdf8ba4548a8931d688c3a8982, because I'm not if we can use Openshift machine API namesapce there.